manuscript/04-Example.md

```diff
@@ -16,20 +16,23 @@ metrics, logging, tracing, traffic routing, circuit breaking, mTLS, and
 authorization. Although Istio is designed to be platform-independent, it
 started with first class support for Kubernetes.
 
-[Figure 4.1](#fig-example-istio) reflects how the service mesh is
+[Figure 4.1](#fig-example-istio) reflects how a service mesh is
 located between the orchestrator (top) and the application
-(bottom). The four core components of Istio make up the control plane:
-Galley, Pilot, Mixer, and Citadel. They communicate with the service
-proxies to distribute configurations, receive recorded network traffic
+(bottom). In Istio version 1.5, the formerly distributed Control Plane 
+was unified into a single process (istiod). It communicates with the service
+proxies to distribute configuration, receive recorded network traffic
 and telemetry data, and manage certificates. Istio uses [*Envoy*](https://www.envoyproxy.io)
 as service proxy, a widely adopted open source proxy that is used by
 other service meshes, too.
 
 ![Figure 4.1: Istio Architecture](images/example-istio.png){#fig-example-istio}
 
-In addition to the typical service mesh control and data plane, Istio also adds infrastructure services. They support the monitoring of microservice applications. Instead of developing its own tools, Istio integrates established applications such as  *Prometheus*, *Grafana*, and *Jaeger* and the service mesh dashboard *Kiali*. The image shows that the Istio control plane interacts with the orchestrator, which today is in most cases Kubernetes.
+In addition to the typical service mesh control and data plane, Istio also adds infrastructure services. They support the monitoring of microservice applications. Instead of developing its own tools, Istio integrates established applications such as *Prometheus*, *Grafana*, and *Jaeger* and the service mesh dashboard *Kiali*. The image shows that the Istio control plane interacts with the orchestrator, which today is in most cases Kubernetes.
 
-In a Kubernetes environment, Istio adds over 20 Custom Resource Definitions (CRDs), which demonstrates the complexity of the Istio API and the richness of configuration options. On the one hand, this allows full customization, but on the other hand it clearly affects the usability. Istio also adds a number of components (marked as Istio Core and integrated components in the figure) to an application which increases technical complexity.
+In a Kubernetes environment, Istio adds over 20 Custom Resource Definitions (CRDs), which demonstrates 
+the complexity of the Istio API and the richness of configuration options. On the one hand, this allows 
```

Comment:
"on the other hand" is twice in the sentence at the end of this line.

Reply:
I dont see it. Do you mean this sentence?
> On the one hand, this allows full customization, but on the other hand it clearly affects the usability.

Reply:
Sorry, my mistake.